### DISCUSSION TITLE
System requirements

### DISCUSSION THREAD

**Discussion by seogrady** [floneum/floneum#221] [Q&A] [answered]
I saw in a few issues discussion around checking if the current system is adequate to run a particular LLM. Is there any progress on this? This could definitely help newcomers. I tried running the sample pirate chat app on my M2 laptop and it was painfully slow. I've had no issues running ollama locally and I'm wondering if it's my setup or I've done something wrong. Is there a suggested minimum spec for local development? Should I try running a different model?

**Comment by ealmloff** [comment, marked as answer]
If you are running on a MacBook enabling the metal feature will speed up the model significantly:
```rust
cargo run --example chat --features metal
```

**Reply by seogrady** [reply]
That did the trick. Thanks @ealmloff 👍

**Comment by hugeugo** [comment]
Hello, Im having issues to install and run floneum... I kind of a noob so I probably don't know something but I get this error trying to opening it from the files I downloaded on the release page :
" zsh: bad CPU type in executable: /Users/[...]/Downloads/floneum-aarch64-apple-darwin/floneum "

**Reply by ealmloff** [reply]
The macos build for floneum is targeting M1-M3 devices that run arm chips. If you are using an older device, you will need to build floneum from source by cloning the repo, [installing rust](https://www.rust-lang.org/tools/install), and building the project by navigating to `floneum/floneum/floneum` and running `cargo build --release`
